### PR TITLE
pkg/u8g2: update version

### PR DESCRIPTION
**Pull request by PeterKietzmann**

### CODE REVIEW

File: pkg/u8g2/patches/0002-u8g2-add-riot-os-interface.patch [hunk 1]
```diff
@@ -4,17 +4,17 @@ Date: Wed, 22 Jun 2016 18:04:31 +0200
 Subject: [PATCH 2/3] u8g2: add riot-os interface.
```
Comment (cgundogan) [inline pkg/u8g2/patches/0002-u8g2-add-riot-os-interface.patch]: I'd prefer to keep this patch untouched and introduce another patch that does the adaptation

**Comment by basilfx** [comment]
Tested it on native and EFM32 (using SSD1306). ACK.

U8g2 is already at commit a02a6df6ac8b4ab6156ed9442b70a31270750c6d, which also works.

File: pkg/u8g2/patches/0002-u8g2-add-riot-os-interface.patch (hunk 1 above)
Comment (basilfx) [inline pkg/u8g2/patches/0002-u8g2-add-riot-os-interface.patch]: The package was also updated, so I can imagine the original patch would not be working due to conflicts.

**Comment by PeterKietzmann** [comment]
> I can imagine the original patch would not be working due to conflicts

That's correct. Rebased and started Murdock
